CMakeLists.txt

```diff
@@ -63,6 +63,9 @@ else()
   message( STATUS "There is no ConfTargets.cmake" )
```

Comment:
please, change the above to unconditionally include ${BUILDER_ROOT}/ConfTargets.cmake